web/pgadmin/settings/__init__.py

```diff
@@ -38,6 +42,14 @@ def get_own_menuitems(self):
                     module="pgAdmin.Settings",
                     callback='show',
                     label=gettext('Reset Layout')
+                ),
+                MenuItem(
```

Comment:
what is this change?

Reply:
Previously, I was unable to fetch the **About pagamin 4** option until I added it, but that's my misunderstanding now I have fixed it.